conda/utils.py

```diff
@@ -75,6 +79,43 @@ def url_path(path):
     return 'file://%s' % path
 
 
+def win_path_to_unix(path, root_prefix=""):
+    """Convert a path or ;-separated string of paths into a unix representation
```

Comment:
This appears to be duplicated in `conda/install.py`. I'd prefer to have it here.